Libraries/Components/ScrollView/ScrollViewStickyHeader.js

```diff
@@ -161,4 +167,24 @@ const styles = StyleSheet.create({
   },
 });
 
-module.exports = ScrollViewStickyHeader;
+class ScrollViewStickyHeaderWrapped extends React.Component<Props, State> {
+  _childRef: ?React.ElementRef<typeof ScrollViewStickyHeader>;
+  setNextHeaderY(y: number) {
+    this._childRef && this._childRef.setNextHeaderY(y);
+  }
+  render(): React.Node {
+    return (
+      <ExperimentalVirtualizedListOptContext.Consumer>
```

Comment:
Some thoughts about how we do experimentation with this. 

I'm hesitant about this approach because we're wrapping a Context around everyone's sticky header. 
I'm thinking we approach this like the component re-factors we've been doing where we create an injection that is up to product teams to include in their bundle. The injection is read then instead of a new prop `experimentalVirtualizedListOpt`. 

See how we're doing Animated refactor: 
* Injection: https://github.com/facebook/react-native/blob/1465c8f3874cdee8c325ab4a4916fda0b3e43bdb/Libraries/Animated/createAnimatedComponentInjection.js
Then in our Bundle initialization we have:
```
import createAnimatedComponentInjection from 'createAnimatedComponentInjection';
import createAnimatedComponent_EXPERIMENTAL from 'createAnimatedComponent_EXPERIMENTAL';
...

  if (<our experiment framework says you're in this test>) {
    createAnimatedComponentInjection.inject(
      createAnimatedComponent_EXPERIMENTAL,
    );
  }
```

What do you think about this approach? I'm happy to import and make these changes

Reply:
> What do you think about this approach? I'm happy to import and make these changes

I didn't quite get you, but am happy to go through your PR.